shared_test.sh

```diff
@@ -68,24 +66,14 @@ function check_on_path() {
 # Pulling is not performed, if we are running locally.
 function pullImages() {
   info "Updating non project images to newest from docker-i.dbc.dk. Errors are ignored."
-  # docker pull ${SOAPUI_IMAGE}
-  docker pull ${VIP_POSTGRES_IMAGE}
-  docker pull ${VIP_CORE_IMAGE}
```

Comment:
If the docker images is never pulled manually, won't we run into problems when these lines are removed?

Reply:
${DOCKER_COMPOSE} pull --ignore-pull-failures ${VIP_POSTGRES_SERVICE} ${WS_SERVICE_GOLD} puller images med udgangs punkt i compose filen